### DISCUSSION TITLE
Fullscreen without gaps

### DISCUSSION THREAD

**Discussion by builtStup1d** [janleigh/dotfiles#29] [Support] [answered]
I want to use super + f to go fullscreen without gaps. How do I do that?

**Comment by janleigh** [comment, marked as answer]
> I want to use super + f to go fullscreen without gaps. How do I do that?

I think the actual fullscreen key is `Super + Alt + F`

**Reply by builtStup1d** [reply]
That's like hitting f11, that way I can't have multiple windows. Anyway to toggle gaps to 0?

**Reply by builtStup1d** [reply]
Got it working by adding this into sxhkdrc

super + shift + g
    bspc config window_gap 16 && sh $HOME/.local/bin/tglbar
